### PR TITLE
fix(test): encrypt refresh_token on db query

### LINKED ISSUE #413: Query encrypted refreshToken.token
When refreshTokens are generated token is [encrypted ](https://github.com/mozilla/fxa-oauth-server/blob/master/lib/db/mysql/index.js#L412). When queried it's not. [here](https://github.com/mozilla/fxa-oauth-server/blob/master/lib/db/mysql/index.js#L426) and [here](https://github.com/mozilla/fxa-oauth-server/blob/master/test/db/index.js#L315).

### PR DESCRIPTION
**Pull request by larissagaulia**
fix #413

### CODE REVIEW

**Comment by vladikoff** [comment]
@seanmonstar r?

File: test/db/index.js [hunk 1]
```diff
@@ -312,7 +321,7 @@ describe('db', function() {
         return db.getAccessToken(token);
       }).then(function (t) {
         assert.equal(t, undefined, 'token deleted');
-        return db.getRefreshToken(refreshToken);
+        return db.getRefreshToken(encrypt.hash(refreshToken));
       }).then(function (t) {
         assert.equal(t, undefined, 'refresh_token deleted');
```
Comment (vladikoff) [inline test/db/index.js]: @seanmonstar without the change above this is always undefined, even if the user was not deleted.

**Comment by seanmonstar** [comment]
r+, travis still yellow

**Comment by vladikoff** [comment]
@larissagaulia thanks!! 👍

**Pull request merged**